packages/flutter_localizations/lib/src/l10n/cupertino_de.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "datePickerHourSemanticsLabelOne": "$hour Uhr",
+  "datePickerHourSemanticsLabelOther": "$hour Uhr",
+  "datePickerMinuteSemanticsLabelOne": "1 Minute",
+  "datePickerMinuteSemanticsLabelOther": "$minute Minuten",
+  "datePickerDateOrder": "dmy",
+  "datePickerDateTimeOrder": "date_time_dayPeriod",
+  "anteMeridiemAbbreviation": "VORM.",
+  "postMeridiemAbbreviation": "NACHM.",
+  "todayLabel": "Heute",
+  "alertDialogLabel": "Benachrichtigung",
+  "timerPickerHourLabelOne": "Stunde",
+  "timerPickerHourLabelOther": "Stunden",
+  "timerPickerMinuteLabelOne": "min",
+  "timerPickerMinuteLabelOther": "min",
+  "timerPickerSecondLabelOne": "s",
+  "timerPickerSecondLabelOther": "s",
+  "cutButtonLabel": "Ausschneiden",
+  "copyButtonLabel": "Kopieren",
+  "pasteButtonLabel": "Einfügen",
```

Comment:
```suggestion
  "pasteButtonLabel": "Einsetzen",
```

![iOS-DE-Einsetzen](https://user-images.githubusercontent.com/9061239/57884562-2ddd9780-7829-11e9-997d-e29f19d33819.PNG)

Reply:
I used the default podcast app to check the German "paste" label for iOS